template/web/app/paymaster-bundler/_components/Vault.tsx

```diff
@@ -11,15 +17,17 @@ export default function Vault() {
       <div className="border border-x-0 border-t-0 border-boat-gold" />
 
       <div className="my-4 flex gap-4">
-        <Item src="/account_abstraction/sword.png" altText="Sword" active={false} />
-        <Item src="/account_abstraction/shield.png" altText="Shield" active={false} />
-        <Item src="/account_abstraction/bow_arrow.png" altText="Bow and Arrow" active={false} />
-        <Item
-          src="/account_abstraction/wing_suit.png"
-          altText="Wing Suit"
-          active={false}
-          amount={4}
-        />
+        {ALL_ITEMS.map((item) => {
+          return (
+            <Item
+              key={item.name}
```

Comment:
perhaps name can be duplicated, in this case maybe appending it with token id is good? `key={`${item.name}-${item.id}`}`